src/functions/template-tags/general.php

```diff
@@ -844,11 +844,11 @@ function tribe_event_featured_image( $post_id = null, $size = 'full', $link = tr
 			$post_id = get_the_ID();
 		}
 
-		$image_html     = get_the_post_thumbnail( $post_id, $size );
+		$image_html     = get_the_post_thumbnail( $post_id, apply_filters( 'tribe_event_featured_image_size', $size ) );
```

Comment:
One other thought (also applies to the second hook you added): could you add `$post_id` as one of the vars we make available to callbacks? That way themers could modify the size only for specific posts, etc.